app/src/main/java/com/github/sdp/mediato/model/Location.java

```diff
@@ -38,4 +41,24 @@ public boolean isValid() {
     public void setValid(boolean valid) {
         this.valid = valid;
     }
+
+    public boolean isInRadius(Location centerLocation) {
+        return calculateDistance(centerLocation) <= RADIUS;

```

Comment:
I think it would be better to have `RADIUS` be a parameter of the function

Reply:
Done in [61e1fca]